packages/flutter/lib/src/services/platform_views.dart

```diff
@@ -343,10 +345,13 @@ class AndroidViewController {
   AndroidViewController._(
```

Comment:
GitHub doesn't let me add comments on the lines above, but doc which params can't be null? Also use `@required`?

Reply:
We decided to move the `@required` to the public function (the constructor is private), see: https://github.com/flutter/flutter/pull/19325#discussion_r203199063

Added null param documentation there.